docs/index.md

```diff
@@ -218,8 +218,8 @@ local TWEEN_FAST = TweenInfo.new(0.5, Enum.EasingStyle.Elastic)
 local TWEEN_SLOW = TweenInfo.new(2, Enum.EasingStyle.Sine)
 
 -- Choose more dramatic styles at low health...
-local style = Computed(function()
-	return if health:get() < 20 then TWEEN_FAST else TWEEN_SLOW
+local style = Computed(function(use, scope)
+	return if use(health) < 20 then TWEEN_FAST else TWEEN_SLOW
```

Comment:
Scope shouldn't be provided as it is never used
Should be scoped

Reply:
done